### DISCUSSION TITLE
Cloudflare's War On Mozilla Opensource Browsers

### DISCUSSION THREAD

**Discussion by PharaohJack** [community/community#60028] [Programming Help]
### Select Topic Area

Product Feedback

### Body

I have tried to access several sites protected by Cloudflare security and have found that Seamonkey and even Firefox are being turned down in attempting to access those sites. Two in particular are the Indeed job-searching site and the Sunflower Bank website. I think that after speaking with a Sunflower official, that obstacle is resolved, but the means of accessing Indeed is almost nonexistent. Cloudflare isn't any better. That's why I'm sending this complaint through github, host of many open-source projects. Maybe someone will have the sense to realize that chasing prospective applicants away by rejecting their longstanding browser and email software is not a good idea. It's gawd-damned dumb.

**Comment by 3urobeat** [comment]
I can access Indeed using Librewolf, a Firefox fork, just fine. Cloudflare might block your request based on a different reason than the browser you are using

**Comment by samus-aran** [comment]
Hi @PharaohJack ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
